its allin. Code with db in AppServer

shadowoffox opened the pull request.

shadowoffox committed:
its allin. Code with db in AppServer

shadowoffox committed:
Поправил

OKrylov commented on src/chat.java:
```diff
@@ -0,0 +1,87 @@
+import java.sql.SQLException;
+
+
+public class chat extends Application implements sendMsg {
```
Классы и интерфейсы именуем с заглавной буквы

OKrylov commented on src/ChatServer.java:
```diff
@@ -0,0 +1,92 @@
+                    if (matcher.matches()) {
+                        String username = matcher.group(1);
+                        String password = matcher.group(2);
+                        if (authService.AuthUser(username, password)) {
```
Методы именуем с прописной буквы

OKrylov reviewed (approved):
1 задание реализовано корректно.
2* задания не увидел, но оно опциональное.

Общие замечания:
1. лучше разбейте на отдельные модули клиента и сервера, это ведь все-таки разные приложения, которые общаются друг с другом, у них слабая связность по общей кодовой базе + будет проще ориентироваться по проекту.
2. На клиенте не отображаются сообщения самого пользователя, так мы не видим всю беседу с другим юзером.

shadowoffox committed:
Добавил отображение сообщения у обоих пользователей, добавил смену логина(но пока не придумал, как изменить на сервере. Добавил приветственное сообщение при входе.)
Не сделал: 1 обновлялся список на сервере. 2 Обновлялся в комбобоксе у пользователей. 3 По папкам пока не раскидал.

shadowoffox committed:
попытки прошлой недели + логгирование чата при выходе.
